server/UMD_utils/UMD_export.py

```diff
@@ -151,14 +151,39 @@ def export_norms_tab(folders, userMap, user):
                                 userDataFound[mapped][normMap[normKey]] = norms[normKey]
                 for key in userDataFound.keys():
                     for normKey in userDataFound[key].keys():
-                        columns = [
-                            key,
-                            videoname,
-                            f'{videoname}_{t["id"]:04}',
-                            normKey,
-                            userDataFound[key][normKey],
-                        ]
-                        writer.writerow(columns)
+                        value = userDataFound[key][normKey]
+                        if value == 'ahered':
+                            value = 'adhere'
+                        if value == 'violated':
+                            value = 'violate'
```

Comment:
Are these necessary? It would be better to make sure these are spelled correctly throughout. Even better than that would be to define an enum type or constants to represent the string values.

Reply:
I'll refactor to string constants to represent the values.  I want to make sure it is backward compatible where existing annotations will output the proper values when exported again as well.  Meaning that 'None' will output 'EMPTY_NA'

Reply:
My specific question in this case was about the `ahered` and `violated` spellings. Do we still require special casing to deal with those or can we remove these lines?

Reply:
To maintain backward compatibility with existing annotations I've modified it so that if the value is violated/violate it will output the correct form of 'violate' for the export functionality.  It's now factored out into a `includes(list)` in case we change the front-end interface to something else.